core/src/test/kotlin/org/stellar/anchor/sep6/Sep6ServiceTestData.kt

```diff
@@ -11,8 +11,17 @@ class Sep6ServiceTestData {
                     "authentication_required": true,
                     "min_amount": 1,
                     "max_amount": 10000,
-                    "funding_methods": ["SEPA", "SWIFT"]
-       
+                    "funding_methods": ["SEPA", "SWIFT"],
+                    "fields": {
+                        "type": {
+                            "description": "type of deposit to make",
+                            "choices": [
+                                "SEPA",
+                                "SWIFT"
+                            ],
+                            "optional": false
```

Comment:
Should `optional` be true now that we accept `funding_method` as well?

Reply:
Can you help clarify what "optional" means in this context? Who is the intended audience, and what is it optional for?

Is it for anchor that this is optional to config? Or is it for wallet to optional past in some request?

Reply:
This field tells the wallet that `type` is an optional parameter for the deposit and withdrawal requests.